rpc/legacyrpc/methods.go

```diff
@@ -3104,6 +3108,141 @@ func signRawTransactions(s *Server, icmd interface{}) (interface{}, error) {
 	return &dcrjson.SignRawTransactionsResult{Results: toReturn}, nil
 }
 
+// startAutoBuyer handles the startautobuyer command.
+func startAutoBuyer(s *Server, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrjson.StartAutoBuyerCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, &ErrUnloadedWallet
+	}
+
+	config := &ticketbuyer.Config{
+		AccountName:             cmd.Account,
+		AvgPriceMode:            s.ticketbuyerConfig.AvgPriceMode,
```

Comment:
Since this is being used for defaults when unset in the request, and since more fields may be added in the future, I would instead make a copy of s.ticketbuyerConfig and then set only those fields which are present in the request.